### PR TITLE
fix(onp): fingerprint updates

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: identity-and-access-management/organizations-and-projects/concepts.mdx [hunk 1]
```diff
@@ -42,8 +42,8 @@ A Scaleway resource is either a product or a feature in the Scaleway Ecosystem.
 
 An SSH key (**S**ecure **Sh**ell key) allows passwordless connection to an [Instance](/compute/instances/concepts/#instance). An SSH key is generated by creating an RSA key pair, consisting of two elements:
 
-- a sensitive **identification key**, (aka private key) which must be stored securely
-- a **public key**, which is uploaded to the Scaleway interface.
+- A **identification key** (also known as a private key), which you must keep securely on the computer you want to connect from.
```
Comment (jcirinosclwy) [inline identity-and-access-management/organizations-and-projects/concepts.mdx:45]:
```suggestion
- An **identification key** (also known as a private key), which you must keep securely on the computer you want to connect from.
```

File: identity-and-access-management/organizations-and-projects/how-to/create-ssh-key.mdx [hunk 2]
```diff
@@ -15,8 +15,8 @@ categories:
 
 [SSH keys](/console/account/concepts/#ssh-key) allow you to securely connect to your Instances, Elastic Metal servers, and Mac minis without using a password. An SSH key consists of a key pair, which has two elements:
 
-- An **identification key** (also known as a private key), which you must keep securely on the computer you want to connect from.
-- A **public key** which you must upload to the Scaleway interface. This is then transferred to your Instance during the boot process.
+- A **identification key** (also known as a private key), which you must keep securely on the computer you want to connect from.
```
Comment (jcirinosclwy) [inline identity-and-access-management/organizations-and-projects/how-to/create-ssh-key.mdx:18]:
```suggestion
- An **identification key** (also known as a private key), which you must keep securely on the computer you want to connect from.
```

File: identity-and-access-management/organizations-and-projects/how-to/create-ssh-key.mdx [hunk 3]
```diff
@@ -114,7 +114,7 @@ On Windows, you can use the third-party application [PuTTYgen](https://www.chiar
     - Enter the same passphrase again to confirm and press Enter.
 
 3. This command will generate two files:
-    - `<key_name>`: The private key file (e.g., `id_rsa`)
+    - `<key_name>`: The private key, file (e.g., `id_rsa`)
```
Comment (jcirinosclwy) [inline identity-and-access-management/organizations-and-projects/how-to/create-ssh-key.mdx:117]: Why the comma here?

**Review by jcirinosclwy — APPROVED** [review]

File: identity-and-access-management/organizations-and-projects/how-to/create-ssh-key.mdx (hunk 3 above)
Comment (ldecarvalho-doc) [reply identity-and-access-management/organizations-and-projects/how-to/create-ssh-key.mdx:117]: dont remember hahaha

Comment (ldecarvalho-doc) [reply identity-and-access-management/organizations-and-projects/how-to/create-ssh-key.mdx:117]:
```suggestion
    - `<key_name>`: The private key file (e.g., `id_rsa`)
```

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**